packages/worker/src/tasks/syncPackageIndexBatch.ts

```diff
@@ -1,7 +1,7 @@
-import { internalApi, PackageMetadata } from '@gradejs-public/shared';
+import { systemApi, SystemAPI, PackageMetadata } from '@gradejs-public/shared';
```

Comment:
Can we use the same identifier for types and runtime? Having two identifiers with different casing looks misleading.

Reply:
fixed f6de5fb